azure/Guidelines.md

```diff
@@ -807,58 +950,78 @@ These are described in the following two sections.
 A POST or DELETE long-running operation accepts a request from the client to initiate the operation processing and returns
 a [status monitor](https://datatracker.ietf.org/doc/html/rfc7231#section-6.3.3) that reports the operation's progress.
 
+<a name="lro-no-post-create"></a>
 :no_entry: **DO NOT** use a long-running POST to create a resource -- use PUT as described below.
 
+<a name="lro-operation-id-request-header"></a>
 :white_check_mark: **DO** allow the client to pass an `Operation-Id` header with an ID for the operation's status monitor.
 
+<a name="lro-operation-id-default-is-guid"></a>
 :white_check_mark: **DO** generate an ID (typically a GUID) for the status monitor if the `Operation-Id` header was not passed by the client.
 
+<a name="lro-operation-id-unique-except-retries"></a>
 :white_check_mark: **DO** fail a request with a `400-BadRequest` if the `Operation-Id` header matches an existing operation unless the request is identical to the prior request (a retry scenario).
 
+<a name="lro-valid-inputs-synchronously"></a>
 :white_check_mark: **DO** perform as much validation as practical when initiating the operation to alert clients of errors early.
 
+<a name="lro-returns-202"></a>
 :white_check_mark: **DO** return a `202-Accepted` status code from the request that initiates an LRO if the processing of the operation was successfully initiated (except for "PUT with additional processing" type LRO).
 
+<a name="lro-returns-only-202"></a>
 :warning: **YOU SHOULD NOT** return any other `2xx` status code from the initial request of an LRO -- return `202-Accepted` and a status monitor even if processing was completed before the initiating request returns.
 
+<a name="lro-returns-status-monitor"></a>
 :white_check_mark: **DO** return a status monitor in the response body as described in [Obtaining status and results of long-running operations](#obtaining-status-and-results-of-long-running-operations).
 
+<a name="lro-returns-operation-location"></a>
 :ballot_box_with_check: **YOU SHOULD** include an `Operation-Location` header in the response with the absolute URL of the status monitor for the operation.
 
+<a name="lro-operation-location-includes-api-version"></a>
 :ballot_box_with_check: **YOU SHOULD** include the `api-version` query parameter in the `Operation-Location` header with the same version passed on the initial request if it is required by the get operation on the status monitor.
 
-:ballot_box_with_check: **YOU SHOULD** allow any valid value of the `api-version` query parameter to be used in the get operation on the status monitor.
-
```

Comment:
This same rule appeared in both the POST/DELETE and PUT sections but really pertains to the status monitor endpoint, so I deleted these two instances and added the rule to the status monitor section.